packages/profile/src/components/TeacherEdit.js

```diff
@@ -28,13 +38,20 @@ export default function TeacherEdit({
   onlyParameterProp,
   isEditable,
   header,
+  nestedHeader,
+  nestedCollapse,
+  seeMore,
+  seeMoreBelowSection,
+  obj,
+  workData,
 }) {
-  const { t } = useTranslation("student");
+  const { t } = useTranslation();
```

Comment:
review if this correct use? do you need to use translation as useTranslation('profile')?